mongodb/flask/README.md

```diff
@@ -0,0 +1,14 @@
+# MongoDB
+This is a basic Flask application that connects to MongoDB and sets up a simple API.
+
+## Getting Started
+
+The server should run automatically on port `5000` when starting a workspace. To run it manually, execute:
+
+```
+./devserver.sh
```

Comment:
When you run the server on `http://127.0.0.1:5000` and open it up in the browser you get a 404.

![Screenshot 2024-10-29 at 4 31 22 PM](https://github.com/user-attachments/assets/58a07411-b6a0-42bd-85e5-685038ba186f)

This is because `http://127.0.0.1:5000/users` is a valid routed but does not have a fallback like the other examples.

Reply:
I'll add a default '/' route.